maintainers/patches

```diff
@@ -0,0 +1,88 @@
+#!/bin/zsh
+
```

Comment:
Don't you want to `set -e` at the beginning to stop the script at the first error?

Reply:
Done. I usually do it but hesitated in this case because sometimes commands return a failing status for normal operation. But I don't think it's the case here (at least not in my experiments).